lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -739,4 +740,46 @@ sub estimated_gtmp_for_instrument_data  {
     );
 }
 
+
+sub get_supersede_merged_alignment_results {
```

Comment:
I would call this `get_superseded_merged_alignment_results`, since it returns results that have been superseded by others rather than results that supersede others.